src/Xabe.FFmpeg/Conversion/Conversion.cs

```diff
@@ -589,8 +642,16 @@ public static IConversion New()
 
         /// <inheritdoc />
         public IConversion UseHardwareAcceleration(HardwareAccelerator hardwareAccelerator, VideoCodec decoder, VideoCodec encoder, int device = 0)
```

Comment:
Why you changed that method?

Reply:
Look at this method 

if (hardwareAccelerator != HardwareAccelerator.software)
            {
                return UseHardwareAcceleration($"{hardwareAccelerator}", decoder.ToString(), encoder.ToString(), device);
            }
            else
            {
                this.UseMultiThread(true);
                return this;
            }

then look at how its called

 .UseHardwareAcceleration(IsGPUEncode ? HardwareAccelerator.qsv: HardwareAccelerator.software, DecoderCodec, Encoder, 0)

otherwise i have two massive if statements depending on if its hardware or software encoding.